deluge/log.py

```diff
@@ -107,7 +107,7 @@ def findCaller(self):  # NOQA
 }
 
 
-def setup_logger(level="error", filename=None, filemode="w"):
+def setup_logger(level="error", filename=None, filemode="w", max_size="50M"):
```

Comment:
I actually agree the function should take an integer, but it doesn't really a need a default value?
I'd say the value should be in bytes as a user may need less than MB even if it sounds strange.
Also considering the convert function outputs in bytes anyways, there is nothing to gain by converting that to MBs?